app/javascript/components/SearchPanel.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import KeyWordSearch from './KeywordSearch';
```

Comment:
For consistency:

```suggestion
import KeywordSearch from './KeywordSearch';
```

Or change the module name to `KeyWordSearch`.  If you go with the class name `KeywordSearch`, please ensure that's updated everywhere.

I lean toward `KeywordSearch` as we naturally write it as "keyword" instead of "key word".  But I defer to you, just as long as the capitalization is internally consistent throughout the code.